scripts/o3de/o3de/repo_properties.py

```diff
@@ -352,7 +391,13 @@ def add_parser_args(parser):
     group = parser.add_argument_group('properties', 'arguments for modifying individual remote repo properties.')
     group.add_argument('--repo-name','-rn',  type=str, required=False,
                        help='The name of the remote repository.')
-    group = parser.add_mutually_exclusive_group(required=False)
+    group.add_argument('--auto-update', '-au', type=str, nargs='*', required=False,
+                       help='Checks for any new Gems/Projects/Templates in the associated directories that have not been added'
+                       ' to your repo.json fields.'
+                       ' Pass in lower case object type as args that should look like this: gem/ project/ template'
+                       ' Note: This does not update the deleted gems/projects/templates, please use'
+                        '--delete-gems if you want to delete data from repo.json file')
```

Comment:
Can we default to updating gems projects and templates if they don't specify the object types?  That seems like it'd be the most convenient/common use.
```suggestion
    group.add_argument('--auto-update', '-au', type=str, nargs='*', default='gem project template', required=False,
                       help='Checks for any new Gems/Projects/Templates in the associated directories that have not been added'
                       ' to your repo.json fields.'
                       ' Updates all object types if no object types are provided.  (e.g. --auto-update). '
                       ' If object types are provided, auto update will  limit the update to those specific object types. e.g. --auto-update gem project  (only update Gems and Projects, not Templates)'
                       ' Note: This does not update the deleted gems/projects/templates, please use'
                        '--delete-gems if you want to delete data from repo.json file')
```